### PR TITLE
Dynamic raw_scale adoptation.

### PR DESCRIPTION
**Pull request by yucedagonurcan**
#17 
Database directory can be set to replace paths in labels file.
Image cropping done dynamically from input image height and width variables.
Added additional visualizations for predictions.
Raw Scaling is now dynamic with respect to IMAGE_MAX_VALUE

### CODE REVIEW

**Comment by yucedagonurcan** [comment]
> #17
> Database directory can be set to replace paths in labels file.
> Image cropping done dynamically from input image height and width variables.
> Added additional visualizations for predictions.
> Raw Scaling now dynamic with respect to IMAGE_MAX_VALUE

File: test_on_dtld/ssd_dtld_test.py [hunk 1]
```diff
@@ -45,6 +46,7 @@
 IMAGE_MAX_VALUE = 255
 SSD_INPUT_IMAGE_WIDTH = 2048
 SSD_INPUT_IMAGE_HEIGHT = 512
+DATABASE_DIR = "/media/user/disk/Datasets/TrafficLightRecognition/DTLD"
```
Comment (julimueller) [inline test_on_dtld/ssd_dtld_test.py:49]: Could you at least add a check that this directory exists?

File: test_on_dtld/ssd_dtld_test.py [hunk 2]
```diff
@@ -320,10 +322,13 @@ def main(args):
                 cv2.rectangle(
                     img_color_orig, (xmin, ymin), (xmax, ymax), (0, 0, 255), 2
                 )
+                cv2.putText(img_color_orig, "RED",(xmin, ymin), cv2.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 255))
```
Comment (julimueller) [inline test_on_dtld/ssd_dtld_test.py:325]: Really needed? I guess the rectangle color is more or less self-explanatory..

File: test_on_dtld/ssd_dtld_test.py [hunk 3]
```diff
@@ -340,6 +347,8 @@ def main(args):
                     (255, 255, 255),
                     2,
                 )
+                cv2.putText(img_color_orig, "UNKOWN", (xmin, ymin), cv2.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 0))
```
Comment (julimueller) [inline test_on_dtld/ssd_dtld_test.py:350]: typo unknown

File: test_on_dtld/ssd_dtld_test.py (hunk 2 above)
Comment (yucedagonurcan) [reply test_on_dtld/ssd_dtld_test.py:325]: Yes you are right, it was just for me. I am used to see tl detections like this.

File: test_on_dtld/ssd_dtld_test.py (hunk 1 above)
Comment (yucedagonurcan) [reply test_on_dtld/ssd_dtld_test.py:49]: You are absolutely right.

**Pull request closed without merging**